lib/tasks/rubocop.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.load_tasks
+
+require 'rubocop/rake_task'
```

Comment:
This doesn't work as-is. You'll need to:

1) Change the extension from `.rb` to `.rake`
2) Change the content to:
```ruby
require 'rubocop/rake_task'

RuboCop::RakeTask.new
```